salt/metalk8s/registry/populated.sls

```diff
@@ -0,0 +1,43 @@
+{% set images = [
```

Comment:
It'd be really good to have this list somewhere else, where the build infrastructure also retrieves it from, etc.